_pages/en/it-picture-long-term.md

```diff
@@ -8,137 +8,25 @@ categories: Work In Progress
 permalink: /it-picture-long-term.html
 ---
 
-## IT Picture - Long Term
+## Long Term IT Picture - 2030
 
-The IT Picture describes the state of IT at ESDC in **long term** to enable its Service Vision (TODO: link to ESDC's Service Vision).
+- [GC Wide](#gc-wide)
+- [IT within ESDC](#it-within-esdc)
 
-- [Modern Open Organization](#modern-open-organization)
-  - [Lean IT Governance](#lean-it-governance)
-  - [Enterprise Architecture (EA)](#enterprise-architecture-ea)
-  - [Continuous Delivery](#continuous-delivery)
-  - [Internal IM/IT services by IITB](#internal-imit-services-by-iitb)
-- [Strong Culture of Learning](#strong-culture-of-learning)
-- [References](#references)
+### GC Wide
 
-### Context - IT in the delivery of ESDC services to citizens
+Building on the [Medium Term IT Picture - 2025](it-picture-medium-term.html), the [Digital Operations Strategic Plan: 2018-2022 Timeline](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html#ToC15) and the [Policy on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603), digital delivery of services and benefits from the GC are automated and integrated in the lives of Canadians.
```

Comment:
Should we add the **Digital Industries Table Challenge** and CDS's response? https://digital.canada.ca/roadmap-2025/

Reply:
Hmm… Dunno, that's external. Could go both ways I guess? :thinking: